Feat/plan checkout options (#79)
Salable/node-sdk: docs/docs/plans/get-checkout-link.md

| allowPromoCode   | The `allowPromoCode` parameter controls the visibility and functionality of the Promotion Code field in your checkout process.<br/>**Stripe**<br/>In Stripe, you can use either `allowPromoCode` or `promoCode`, but not both simultaneously. Setting `allowPromoCode` to true displays the promo code field, allowing customers to enter a code.<br/>_Note_: If you would prefer to use a pre-filled specific promo code, use the `promoCode` parameter with the Promotion Code ID from your Stripe dashboard. Remember, using `promoCode` in Stripe means customers cannot modify the code during checkout.<br/>**Paddle**<br/>By default, this is set to true. Setting `allowPromoCode` to false will hide the promo code field in the checkout. If you're using the `promoCode` parameter to pre-fill a code, the user can still modify it during checkout.                                                                                                                         |      ❌      |

> | currency         | Currency shortname of the currency to be used in the checkout. The currency must be added to the plan's product in Salable. Default value is the default currency's shortname on the plan's product.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |      ❌      |

Reviewer: ```suggestion
| currency         | Shortname of the currency to be used in the checkout. The currency must be added to the plan's product in Salable. If not specified, it defaults to the currency selected on the product.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |      ❌      |
```
